src/renderers/shaders/ShaderChunk/fog_pars_vertex.glsl

```diff
@@ -0,0 +1,3 @@
+#ifdef USE_FOG
+  varying float fogDepth;
+#endif
```

Comment:
I was referring to the lack of a new line at end of your glsl files... but do appreciate the improved formatting, though. :)